### DISCUSSION TITLE
Is there a release schedule for the next dpc++ version?

### DISCUSSION THREAD

**Discussion by RealFYang** [intel/llvm#5586] [Q&A]
Hi,
    I see the latest dpc++ release is: oneAPI DPC++ Compiler 2021-09. 
    Is there a release schedule for the next dpc++ version? I don't know where to find this kind of infomation.
    Many thanks.

**Comment by alexbatashev** [comment]
@RealFYang official releases are published on [intel.com](https://www.intel.com/content/www/us/en/developer/tools/oneapi/overview.html#gs.pt63fx), all the publicly available information on upcoming releases is also communicated through that channel.

**Reply by RealFYang** [reply]
@alexbatashev : Thanks for the reply :-)
To be more specific, I am asking about releases here: https://github.com/intel/llvm/releases/tag/2021-09 
I am assuming there will be similar release notes here on https://github.com/intel/llvm/releases/tag/ for the next release (maybe 2022-01?). Am I right?  If true, when will that happen? Thanks again.

**Reply by alexbatashev** [reply]
@RealFYang release notes are provided in https://github.com/intel/llvm/blob/sycl/sycl/ReleaseNotes.md for all the past releases. There will be a tag and release notes update on GitHub, but we do not provide announcements for future releases beforehand.

**Reply by RealFYang** [reply]
@alexbatashev : I see. Thanks for letting me know.
